eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -466,34 +459,27 @@ class EclairRpcClient(val instance: EclairInstance)(
     }
   }
 
-  private def getPayload(response: HttpResponse): Future[JsValue] = {
-    val payloadF = response.entity.dataBytes.runFold(ByteString.empty)(_ ++ _)
-
-    payloadF.map { payload =>
-      val parsed: JsValue = Json.parse(payload.decodeString(ByteString.UTF_8))
-      parsed
+  private def sendRequest(req: Request[String, Nothing]): Future[JsValue] = {
+    import com.softwaremill.sttp.asynchttpclient.future.AsyncHttpClientFutureBackend
+    implicit val backend: SttpBackend[Future, Nothing] =
+      AsyncHttpClientFutureBackend()
+    req.send().map { response =>
+      Json.parse(response.unsafeBody)
     }
   }
 
-  private def sendRequest(req: HttpRequest): Future[HttpResponse] = {
-    val respF = Http(m.system).singleRequest(req)
-    respF
-  }
-
   private def buildRequest(
       instance: EclairInstance,
       methodName: String,
-      params: (String, String)*): HttpRequest = {
+      params: (String, String)*): Request[String, Nothing] = {
```

Comment:
We seem to use `Request[String, Nothing]` a lot now and it isn't really clear what that means from the name, I think we should use a type alias for this with a better name